docs/xxjitserveruseaotcache.md

```diff
@@ -23,33 +23,34 @@
 
 # -XX:\[+|-\]JITServerUseAOTCache
 
-This option enables the caching of AOT-compiled methods in the JITServer server.
+This option enables or disables the caching of AOT-compiled methods in the JITServer server.
 
 ## Syntax
 
         -XX:[+|-]JITServerUseAOTCache
 
 | Setting                    | Effect  | Default                                                                              |
 |----------------------------|---------|:------------------------------------------------------------------------------------:|
-|`-XX:+JITServerUseAOTCache` | Enable  |                                                                                      |
-|`-XX:-JITServerUseAOTCache` | Disable | :fontawesome-solid-check:{: .yes aria-hidden="true"}<span class="sr-only">yes</span> |
+|`-XX:+JITServerUseAOTCache` | Enable  | :fontawesome-solid-check:{: .yes aria-hidden="true"}<span class="sr-only">yes</span> |
+|`-XX:-JITServerUseAOTCache` | Disable |                                                                                      |
 
 ## Explanation
 
 When you enable this option, the JITServer server caches the AOT-compiled methods. When a JITServer client requests an AOT compilation and the requested method exists in the cache, the server does not have to recompile the method. This feature, therefore, improves the CPU utilization of the JITServer technology and of the cluster.
 
- :fontawesome-solid-pencil:{: .note aria-hidden="true"} **Note:** This option has the following requirements:
+ :fontawesome-solid-pencil:{: .note aria-hidden="true"} **Note:** This option must be enabled both at the client JVM and at the server. Although this option is by default enabled at the server but it is still disabled for the JITServer clients. That means that you don't have to specify the `-XX:+JITServerUseAOTCache` option at the server to enable the JITServer AOT caching feature, but you must specify this option for each client who wants to use this feature.
```

Comment:
"Although this option is by default enabled at the server but it is still disabled for the JITServer clients."
-->
"Although this option is by default enabled at the server, it is still disabled for the JITServer clients."